content/dev/docs/timemgr/index.md

```diff
@@ -1,3 +1,14 @@
+---
```

Comment:
The file has to be named `_index.md` not `index.md` to be a section root.

I'm not sure though that every time manager needs a separate subpage. So far we don't have third-level documents at all, I don't know whether it looks fine in the menu. (but it's up to you)

Reply:
have moved the page to play/configuration for now. It is only created as a draft page so another change is still needed.